books/free-programming-books-he.md

```diff
@@ -0,0 +1,36 @@
+### Index
+
+* [C#](#cs)
+* [Assembly](#assembly)
+* [Python](#python)
+* [ללא תלות בשפה](#ללא-תלות-בשפה)
+  * [מערכות הפעלה](#מערכות-הפעלה)
+  * [רשתות](#רשתות)
+
+
+### Assembly
+
+* [ארגון המחשב ושפת סף](https://data.cyber.org.il/python/python_book.pdf) – ברק גונן, המרכז לחינוך סייבר (PDF)
+
+
+### <a name="cs">C#</a>
```

Comment:
- Usually, an anchor that represent an alias has no content.
 - Also apply rename in TOC: `c-sharp` 

```suggestion
### <a name="c-sharp"></a>C#
```